AEPAnalytics/Sources/Analytics.swift

```diff
@@ -444,6 +463,7 @@ public class Analytics: NSObject, Extension {
         let locale = ServiceProvider.shared.systemInfoService.getActiveLocaleName()
         if !locale.isEmpty {
             headers[AnalyticsConstants.HttpConnection.HEADER_KEY_ACCEPT_LANGUAGE] = locale
+            analyticsProperties.locale = locale
```

Comment:
Where else do we use locale?

Reply:
As per the discussion, removing locale since we use it just here and would get it from systemInfoService.